CMakeLists.txt

```diff
@@ -202,39 +205,19 @@ foreach(file ${SQL_FILES})
 	configure_file(${CMAKE_SOURCE_DIR}/migrations/cdserver/${file} ${PROJECT_BINARY_DIR}/migrations/cdserver/${file})
 endforeach()
 
+# Add system specfic includes for Apple, Windows and Other Unix OS' (including Linux)
+if (APPLE)
+	include_directories("/usr/local/include/")
+endif()
```

Comment:
Is there a particular reason this comment mentioned 3 other OS' but doesnt actually include anything specific?

Reply:
I think this just appears as new but was moved? In general we should not include /usr/local

Reply:
Ah yeah this is in the old version.  Not going to touch it for now, but will just comment that this likely shouldnt be done.